_posts/2017-07-28-reproducible-research-badges.md

```diff
@@ -0,0 +1,154 @@
+
+https://img.shields.io/badge/executable-yes-44cc11.svg
+
+Badges for reproducibility, peer review status and license are color coded to provide additional visual aids indicating for example (un)successful reproducibility.
+
+While a common procedure is reused across the badge types, other badges are generated with different functions and get their information from other sources. The information for peer review and releasetime badges is requested from the external services DOAJ and Crossref. The spatial badge also uses the o2r services, but it additionally converts the spatial information from coordinates into textual information, i.e. place names, using the [Geonames API](http://www.geonames.org/export/web-services.html). 
+
+## Spread badges over the web
+
+So there is a great badge server, and databases providing manifold badge information, but how to get them displayed online? The sustainable way would be for research website operators to agree on a common badge system and design, and then incorporate these badges on their platforms. But we know that is never ever going to happen, so instead of waiting we created a chrome extension for common research websites. The o2r-extender (https://github.com/o2r-project/o2r-extender) automatically inserts badges into search results or publication pages using client-side browser scripting. Its available in the Chrome Web Store (*[here](https://chrome.google.com/webstore/detail/opening-reproducible-rese/fhhfncpkfohlhphlcgpkbpialfhkmbil)*)  and ready to be tried out.
+
+The extender currently supports the following websites:
+
+- Google Scholar (https://scholar.google.de/)
+- DOAJ.org (https://doaj.org/)
+- ScienceDirect.com (http://www.sciencedirect.com/)
+- ScienceOpen.com (https://scienceopen.com/)
+- PLOS.org (https://www.plos.org/)
+- Microsoft Academic (https://academic.microsoft.com/)
+- Mendeley (https://www.mendeley.com/)
+
+For each listed article contained in these research websites, the extender requests a set of badges from the o2r-badger. These are then inserted into the page’s HTML code after rendering the regular website:
+
+![google scholar badges](/public/images/2017-07-28-badges/google_scholar_badges.png "Figure 3: Badges integrated into Google Scholar search results")
+<p class="attributionInlineImage">Figure 3: Badges integrated into Google Scholar search results</p>
+
+
+When the badger does not find information for a certain DOI, it returns a grey “not available” - badge instead, as shown in the screenshot above for the outermost license and peer review badges. 
+
+The extender consists of a content script, similar to a [userscript](http://techsupportguides.com/what-is-a-userscript/), for each research website. The content scripts insert a set of badges (Figure 3) for each article in the respective website. They all use a set of base functions defined in the chrome extension for generating HTML, getting DOIs and inserting badges.
```

Comment:
"Chrome" again.

Also, since this is partly a quite technical blog post, please add a couple of links to the actual source code of the scripts.